mm2src/mm2_main/src/rpc.rs

```diff
@@ -301,6 +304,8 @@ async fn rpc_service(req: Request<Body>, ctx_h: u32, client: SocketAddr) -> Resp
     Response::from_parts(parts, Body::from(body_escaped))
 }
 
+// TODO: This should exclude TCP internals, as including them results in having to
+// handle various protocols within this function.
 #[cfg(not(target_arch = "wasm32"))]
 pub extern "C" fn spawn_rpc(ctx_h: u32) {
```

Comment:
Can we add doc  com for spawn_rpc for both targets

Reply:
They will be changed a lot tho, as I mention in the TODO note